benchmarks/benchmark_sed.py

```diff
@@ -188,6 +188,7 @@ def test_loader_compute(loader: BaseLoader) -> None:
         loaded_dataframe, _, loaded_metadata = loader.read_dataframe(
             runs=runs[loader_name],
             collect_metadata=False,
+            force_recreate=True,
```

Comment:
This should not work anymore for all other loaders when #466 is merged.

Reply:
Yes, but this is a keyword specific to Flash so I don't think it'll be passed when using other loaders. Or do you mean the check happens earlier?

Reply:
Like you defined it, it will be passed to all readers. Right now it is being ignored, but will raise after merging #466